net-analyzer/zeek/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<!-- maintainer-needed -->
+    <use>
+        <flag name="zeekctl">Enable ZeekControl, a tool for operating Zeek installations</flag>
```

Comment:
You can sort these for better readability too.

Reply:
Done